src/Text/Pandoc/Writers/Markdown/Inline.hs

```diff
@@ -283,6 +344,7 @@ inlineToMarkdown opts (Span attrs ils) = do
              _ -> id
          $ case variant of
                 PlainText -> contents
+                Markua -> "[" <> contents <> "]" <> attrsToMarkua attrs
```

Comment:
Is this right?  Markua documentation says to use backticks for spans?
(I'm no expert, just glancing at it.)

Reply:
Yes, you are correct, that must have slipped through our checks and is now corrected